style.css

```diff
@@ -0,0 +1,179 @@
+body {
+  margin: 0;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
+  color: #fff;
+  text-decoration: none;
+}
+
+.apple-main {
+  background-image: url("images/main.jpg");
+  background-repeat: no-repeat;
+  width: 100%;
+  height: 590px;
+  background-position: center;
+  background-size: cover;
+  position: relative;
+  margin-top: -5px;
+}
+
+.main-navigation {
+  position: relative;
+  top: 50px;
+}
+
+.main-navigation ul {
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  margin-top: 5px;
+}
+
+.footer-images li {
+  padding-right: 2px;
+  width: 25%;
+}
+
+.main-navigation ul li {
+  box-shadow: 0 1px 2px rgb(0, 0, 0, 0.3);
+  padding-left: 35px;
+  padding-right: 35px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  text-transform: capitalize;
+  background-image: linear-gradient(#333, #5c5c5c);
```

Comment:
[Optional] try this gradient instead
background-image: linear-gradient(#9c9c9c 20%,#5c5c5c, #4c4c4c);